src/AlertBox/AlertBox.jsx

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PureComponent, PropTypes} from "react";
+
+import "./AlertBox.less";
+import {CloseIcon, WarningIcon, SuccessIcon, ErrorIcon, InfoIcon} from "./Icons";
+
+/*
+ * AlertBox is a closable, highlighted box
+*/
+export default class AlertBox extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {isOpen: true};
+  }
+
+  closeBox() {
+    this.setState({isOpen: false});
+  }
+
+  render() {
+    const {children, className, type, title, isClosable} = this.props;
+    const {isOpen} = this.state;
+    const {Icons, cssClass} = AlertBox;
+    if (!isOpen) {
+      return null;
+    }
+    return (
+      <div className={classnames(`AlertBox--${type}`, cssClass.CONTAINER, className)}>
+        <div className={cssClass.HEADER}>
+          <span>{Icons[type]()}</span>
```

Comment:
Only one change I'd make here—just to make clear that this is actually creating a React element, can you in the code above say:

```
const Icon = Icons[type];
```

and then invoke it here as:

```
<span><Icon /></span>
```

That way it's obvious that `Icon` is a React component.